### PR TITLE
Telegraf: fix Windows install instructions.

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #4935

For details, see original #4935

### CODE REVIEW

File: content/telegraf/v1.26/install.md [hunk 1]
```diff
@@ -235,16 +235,22 @@ brew update
 brew install telegraf
 ```
 
-To have launchd start telegraf at next login:
+Run one of the following commands to start Telegraf collecting and processing metrics:
```
Comment (sanderson) [inline content/telegraf/v1.26/install.md:238]:
```suggestion
Run one of the following commands to start Telegraf and begin collecting and processing metrics:
```

File: content/telegraf/v1.26/install.md [hunk 2]
```diff
@@ -235,16 +235,22 @@ brew update
...
+
+To have launchd start Telegraf as a service after the next login:
+
 ```
 ln -sfv /usr/local/opt/telegraf/*.plist ~/Library/LaunchAgents
 ```
-To load telegraf now:
+
+To start Telegraf as a service now:
```
Comment (sanderson) [inline content/telegraf/v1.26/install.md:246]:
```suggestion
Start Telegraf as a launchd service:
```

File: content/telegraf/v1.26/install.md [hunk 3]
```diff
@@ -286,43 +292,72 @@ In PowerShell _as an administrator_, do the following:
...
+
+        ```powershell
+        cd "C:\Program Files\InfluxData\telegraf";
+        mv .\telegraf-{{% latest-patch %}}\telegraf.* .
+        ```
+
+    - **Or**, create a [Windows symbolic link (Symlink)](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/) for
+    `C:\Program Files\InfluxData\telegraf` that points to the extracted directory.
+
+    > The remaining instructions assume that `telegraf.exe` and `telegraf.conf` files are stored in `C:\Program Files\InfluxData\telegraf` or that you created a Symlink to point to this directory.
```
Comment (sanderson) [inline content/telegraf/v1.26/install.md:316]:
```suggestion
    {{% note %}}
The remaining instructions assume that `telegraf.exe` and `telegraf.conf` files are stored in
`C:\Program Files\InfluxData\telegraf` or that you created a Symlink to point to this directory.
    {{% /note %}}
```

File: content/telegraf/v1.26/install.md [hunk 4]
```diff
@@ -286,43 +292,72 @@ In PowerShell _as an administrator_, do the following:
...
+
+3.  Optional: Disable the [`inputs.processes` plugin](/{{% latest "telegraf" %}}/plugins/#input-processes).
```
Comment (sanderson) [inline content/telegraf/v1.26/install.md:318]:
```suggestion
3.  _(Optional)_ Disable the [`inputs.processes` plugin](/telegraf/v1.26/plugins/#input-processes).
```

File: content/telegraf/v1.26/install.md [hunk 5]
```diff
@@ -286,43 +292,72 @@ In PowerShell _as an administrator_, do the following:
...
+    # This plugin ONLY supports non-Windows
+    # [[inputs.processes]]
+    ...
+    #  # use_sudo = false
+    ...
     ```
 
-    Or create a [Windows symbolic link (Symlink)](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/)
-    to point to this directory.
+4.  Optional: Enable a plugin to collect Windows-specific metrics--for example, uncomment the [`inputs.win_services`  plugin](/{{% latest "telegraf" %}}/plugins/#input-win_services) configuration line:
```
Comment (sanderson) [inline content/telegraf/v1.26/install.md:331]:
```suggestion
4.  _(Optional)_ Enable a plugin to collect Windows-specific metrics--for example, uncomment the [`inputs.win_services`  plugin](/telegraf/v1.26/plugins/#input-win_services) configuration line:
```

File: content/telegraf/v1.27/install.md [hunk 6]
```diff
@@ -235,16 +235,22 @@ brew update
 brew install telegraf
 ```
 
-To have launchd start telegraf at next login:
+Run one of the following commands to start Telegraf collecting and processing metrics:
```
Comment (sanderson) [inline content/telegraf/v1.27/install.md:238]:
```suggestion
Run one of the following commands to start Telegraf and begin collecting and processing metrics:
```

File: content/telegraf/v1.27/install.md [hunk 7]
```diff
@@ -235,16 +235,22 @@ brew update
...
+
+To have launchd start Telegraf as a service after the next login:
+
 ```
 ln -sfv /usr/local/opt/telegraf/*.plist ~/Library/LaunchAgents
 ```
-To load telegraf now:
+
+To start Telegraf as a service now:
```
Comment (sanderson) [inline content/telegraf/v1.27/install.md:246]:
```suggestion
Start Telegraf as a launchd service:
```

File: content/telegraf/v1.27/install.md [hunk 8]
```diff
@@ -286,43 +294,72 @@ In PowerShell _as an administrator_, do the following:
...
+
+        ```powershell
+        cd "C:\Program Files\InfluxData\telegraf";
+        mv .\telegraf-{{% latest-patch %}}\telegraf.* .
+        ```
+
+    - **Or**, create a [Windows symbolic link (Symlink)](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/) for
+    `C:\Program Files\InfluxData\telegraf` that points to the extracted directory.
+
+    > The remaining instructions assume that `telegraf.exe` and `telegraf.conf` files are stored in `C:\Program Files\InfluxData\telegraf` or that you created a Symlink to point to this directory.
```
Comment (sanderson) [inline content/telegraf/v1.27/install.md:318]:
```suggestion
    {{% note %}}
The remaining instructions assume that `telegraf.exe` and `telegraf.conf` files are stored in
`C:\Program Files\InfluxData\telegraf` or that you created a Symlink to point to this directory.
    {{% /note %}}
```

File: content/telegraf/v1.27/install.md [hunk 9]
```diff
@@ -286,43 +294,72 @@ In PowerShell _as an administrator_, do the following:
...
+
+3.  Optional: Disable the [`inputs.processes` plugin](/{{% latest "telegraf" %}}/plugins/#input-processes).
```
Comment (sanderson) [inline content/telegraf/v1.27/install.md:320]:
```suggestion
3.  _(Optional)_ Disable the [`inputs.processes` plugin](/{{% latest "telegraf" %}}/plugins/#input-processes).
```

File: content/telegraf/v1.27/install.md [hunk 10]
```diff
@@ -286,43 +294,72 @@ In PowerShell _as an administrator_, do the following:
...
+    # This plugin ONLY supports non-Windows
+    # [[inputs.processes]]
+    ...
+    #  # use_sudo = false
+    ...
     ```
 
-    Or create a [Windows symbolic link (Symlink)](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/)
-    to point to this directory.
+4.  Optional: Enable a plugin to collect Windows-specific metrics--for example, uncomment the [`inputs.win_services`  plugin](/{{% latest "telegraf" %}}/plugins/#input-win_services) configuration line:
```
Comment (sanderson) [inline content/telegraf/v1.27/install.md:333]:
```suggestion
4.  _(Optional)_ Enable a plugin to collect Windows-specific metrics--for example, uncomment the [`inputs.win_services`  plugin](/{{% latest "telegraf" %}}/plugins/#input-win_services) configuration line:
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Looks good. Just some minor suggestions.

File: content/telegraf/v1.26/install.md (hunk 4 above)
Comment (jstirnaman) [reply content/telegraf/v1.26/install.md:318]: https://developers.google.com/style/procedures#optional-steps

**Pull request merged**